app/fonts/stylesheets/home.scss

```diff
@@ -0,0 +1,21 @@
+@font-face {
+    font-family: 'fairviewregular';
+    src: url('fairview_regular-webfont.eot');
+    src: url('fairview_regular-webfont.eot') format('embedded-opentype'),
+         url('fairview_regular-webfont.woff2') format('woff2'),
+         url('fairview_regular-webfont.woff') format('woff'),
+         url('fairview_regular-webfont.ttf') format('truetype'),
+         url('fairview_regular-webfont.svg') format('svg');
+    font-weight: normal;
+    font-style: normal;
+}
+
+.landing-title {
+  color: red;
+  margin-top: 75px;
```

Comment:
Properties should be ordered color, font-size, margin-left, margin-right, margin-top, text-align, width